app/renderer/components/Inspector/Inspector.js

```diff
@@ -123,6 +129,9 @@ export default class Inspector extends Component {
         <Tooltip title="Search for element">
            <Button id='searchForElement' icon="search" onClick={showLocatorTestModal}/>
         </Tooltip>
+        <Tooltip title="Copy XML Source to Clipboard">
+           <Button id='btnSourceXML' icon="copy" onClick={() => this.copySourceXMLToClipboard()}/>
```

Comment:
since `copySourceXMLToClipboard` has effectively only one line of code in it and because it's used only once, might as well just call `copy` here